pandas/tests/frame/test_subclass.py

```diff
@@ -573,3 +575,117 @@ def test_subclassed_boolean_reductions(self, all_boolean_reductions):
         df = tm.SubclassedDataFrame({"A": [1, 2, 3], "B": [4, 5, 6], "C": [7, 8, 9]})
         result = getattr(df, all_boolean_reductions)()
         assert isinstance(result, tm.SubclassedSeries)
+
```

Comment:
can you paramaterize any of this?

Reply:
I don't believe pytests allows you to use fixtures (boolean_reductions, numeric_reductions) in a paramaterized test the way I would need. 

So there's two ways I can see to do this. Paramaterize the two reduction tests into one or create a new fixture combining `boolean_reductions` and `numeric_reductions`. Since paramaterizing the test would require creating another point of maintenance I created `all_reductions`. 

This would help reduce the number of written tests in other files as well, for example look at [pandas/tests/extension/base/reduce.py](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/extension/base/reduce.py)